Sources/Apollo/LegacyCacheReadInterceptor.swift

```diff
@@ -83,6 +99,12 @@ public class LegacyCacheReadInterceptor: ApolloInterceptor {
       }
     }
   }
+
+  private func isDataStillFresh(withAge dataAge: Date, accordingTo ttl: TimeInterval?) -> Bool {
```

Comment:
Minor: `isDataStillValid(for dataAge: Date, accordingTo ttl: TimeInterval?)`

Reply:
🤦 You're right, that's so much better